src/utils.ts

```diff
@@ -0,0 +1,32 @@
+import {IScore} from "./types";
+
+const SCIENCE_KEYS = [
```

Comment:
It is better to specify type of the variable directly instead of forcing cast to type you need:
```
const SCIENCE_KEYS: (keyof IScore)[];
```
Also I noticed that you use *keyof IScore* a lot, so it might be useful to extract this to separate type and use it:
```
export const ScoreKeys = keyof IScore;
```

Reply:
Fixed